### DISCUSSION TITLE
JUG Meeting the June 2021

### DISCUSSION THREAD

**Discussion by Llewellynvdm** [joomengine/Joomla-Component-Builder#754] [Announcements]
## You are all welcome

# Topic (possible)
- Extension Distributor

# Actions
- Help us [select the time](https://doodle.com/poll/ahx42m9mrdsz2vp8) of the next JUG meeting.
- Make more topics suggestions below.

**Comment by marcodings** [comment]
No explicit preference

**Reply by Llewellynvdm** [reply]
Not sure we can do that topic.... `no explicit preference` we _generally_ would **generally** avoid these topics, as it will expose my bios. Not that I mind seeing the BASH is often my shell by default, yet the EFI firmware on some OS's is not that preferred in a implicit, but best to be in a more explicit mode.

:laughing: Above it just me also making fun, thank Marco :tada: At least now we know you just like JCB, even if the content borer you to tears at times.

**Comment by MastersOfMedia** [comment]
Any Monday, Tuesday or Wednesday night is fine.

**Reply by Llewellynvdm** [reply]
Please just bump this doodle with you choice ;)
https://doodle.com/poll/ahx42m9mrdsz2vp8

**Reply by saman222** [reply]
> * Make more topics suggestions below.

hi @Llewellynvdm  and thank you for JCB
would you like talk about language management in JCB? or can you help JCB users about check his component?

**Comment by TLWebdesign** [comment]
Community support in GLIP. I find not many people are active in there. Would be nice to have a thriving community support. Also another topic suggestion: code snippet library. A place where we could share code snippets like handy helper functions or more elaborate things. Idk about other peoples willingness to share stuff like that but i'm up for it. 

Also a "doctor JCB" thing like "doctor Joomla" something that i see in other jugs. People having issues doing things with JCB can get direct help. I think this would be nice as a sort of "after party" So it won't cut into the 1 hour we have. But there is still a place to ask questions in person and show issues in person.

**Reply by cpaschen** [reply]
Just a note about GLIP. Since Ring Central aquired GLIP and rolled it all together in their service, because I also have RingCentral for VOIP, I can't access any other GLIP groups other than my own business groups.
If someone knows how to make it work (that has GLIP and RinCentral VOIP) please let me know.
The RingCentral support was unable to get it to work.

**Reply by cpaschen** [reply]
And @TLWebdesign  that might we why some aren't active in GLIP. That's why I'm not there any longer.

**Comment by MastersOfMedia** [comment]
'Doctor JCB' is nice at te end of the meeting

**Comment by Llewellynvdm** [comment]
# MEETING TIME ANNOUNCEMENT!!

7th of June 2021 <a href="https://doodle.com/poll/ahx42m9mrdsz2vp8" target="_blank" style="color: rgb(0, 146, 255);">see time</a> all welcome at <a href="https://meet.google.com/kxr-nfbv-dww" target="_blank" style="color: rgb(0, 146, 255);">room</a>.

> Please add this event to your <a target="_blank" style="color: rgb(0, 146, 255);" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=MWd2ZG41OGo0bWswMGZuaHIxMTg3NW9oczMgbGxld2VsbHluQHZkbS5pbw&amp;tmsrc=llewellyn%40vdm.io">calendar</a>, and invite more JCB/Joomla developer... we really want to see you there!

**Reply by saman222** [reply]
thank you @Llewellynvdm 
can you send recorded file of this meeting to youtube? (such as first JUG)

**Reply by Llewellynvdm** [reply]
at the moment we are not yet recording it, but we plan to... just not yet sure when. We need a volunteer that will actually manage that for us. I can give them the guidance how to do it... but I can't also manage that.

**Reply by saman222** [reply]
is it possible i start manage that for you in experimental period? i think that i can help on this subject.
